apps/web/.storybook/main.ts

```diff
@@ -1,3 +1,3 @@
-import config from "storybook-helpers/main";
+import config from "@gc-digital-talent/storybook-helpers/main";
 
-export default config;
+export default { ...config };
```

Comment:
While this wasn't breaking anything, there was an annoying warning about re-exporting as default. Deconstruction avoids this warning.